.storybook/preview.tsx

```diff
@@ -4,7 +4,7 @@ import { dark, light } from "./frontifyTheme";
 export const parameters = {
     options: {
         storySort: {
-            order: ["Foundation", "Components"],
```

Comment:
Foundation is now at the end and only contains "Icon". I think we should either put "Icon" into "Components" else or keep "Foundation" at a higher level in the order.

![Screenshot 2022-02-02 at 17 06 34](https://user-images.githubusercontent.com/3276065/152191339-831514dd-0798-40bc-850b-e54b7eaf391a.png)

Reply:
we decided to do that in a separated PR